ttg/ttg/util/dot.h

```diff
@@ -41,9 +43,18 @@ namespace ttg {
       for (auto in : tt->get_inputs()) {
         if (in) {
           if (count != in->get_index()) throw "ttg::Dot: lost count of ins";
+          if(disable_type)
```

Comment:
need to clang-format this ... see https://github.com/TESSEorg/ttg/blob/master/doc/dox/contrib/Workflow.md

Reply:
in general follow the established indentation pattern